metadata.json

```diff
@@ -13,6 +13,14 @@
     "profiles"
   ],
   "dependencies": [
+    {
```

Comment:
We provide example code (eg the nfs HOWTO) that uses data types from simplib and stdlib. If deps aren't declared in the metadata.json, the types won't be found and compilation will fail. 

This just makes that process a little easier.